File: DarkOrbit 10.0/Net/netty/commands/EquipReadyCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ow.Utils;

namespace Ow.Net.netty.commands
{
    class EquipReadyCommand
    {
        public const short ID = 6949;

        public static byte[] write(bool ready)
        {
            ByteArray param1 = new ByteArray(ID);
            param1.writeBoolean(ready);
            return param1.ToByteArray();
        }
    }
}

File: DarkOrbit 9.0/DarkOrbit 9.0 Emulator/src/simulator/netty/serverCommands/EquipReadyCommand.java
package simulator.netty.serverCommands;

import java.io.DataOutputStream;
import java.io.IOException;

import simulator.netty.ServerCommand;

public class EquipReadyCommand
        implements ServerCommand {

    public EquipReadyCommand(boolean param1) {
        this.ready = param1;
    }

    public static int     ID    = 439;
    public        boolean ready = false;

    public void write(DataOutputStream param1) {
        try {
            param1.writeShort(ID);
            this.writeInternal(param1);
        } catch (IOException e) {
        }
    }

    protected void writeInternal(DataOutputStream param1) {
        try {
            param1.writeBoolean(this.ready);
            param1.writeShort(-19324);
        } catch (IOException e) {
        }
    }
}